gapic/templates/%namespace/%name_%version/%sub/services/%service/async_client.py.j2

```diff
@@ -207,6 +207,13 @@ class {{ service.async_client_name }}:
             {% endif %}
         r"""{{ method.meta.doc|rst(width=72, indent=8) }}
 
+        .. code-block::
```

Comment:
Should the `.. code-block::` be inside the if statement a few lines below?

Reply:
Ahh good catch, this would have created a lot of noise in the regen PRs.